packages/core/src/conversation/ConversationService/ConversationService.ts

```diff
@@ -367,6 +367,10 @@ export class ConversationService {
   }
 
   public async wipeMLSConversation(groupId: string): Promise<void> {
+    const isMLSConversationEstablished = await this.isMLSConversationEstablished(groupId);
+    if (!isMLSConversationEstablished) {
+      return;
+    }
```

Comment:
We want to make sure the group exists before wiping it (otherwise an error is thrown)